apps/dot_ext/forms.py

```diff
@@ -88,27 +90,6 @@ class Meta:
     def clean(self):
         client_type = self.cleaned_data.get("client_type")
         authorization_grant_type = self.cleaned_data.get("authorization_grant_type")
```

Comment:
It doesn't seem like we need these vars to be set either now that the rest of the logic here is removed.
```suggestion
```

Reply:
Yep! I'd second this. I think this was meant to get ripped out before. I'd say it's good to go otherwise.